saleor/graphql/product/tests/mutations/test_product_create.py

```diff
@@ -883,13 +884,12 @@ def test_create_product_with_negative_weight(
     }
 
     response = staff_api_client.post_graphql(
-        query, variables, permissions=[permission_manage_products]
+        query,
+        variables,
+        permissions=[permission_manage_products],
+        check_no_permissions=False,
     )
-    content = get_graphql_content(response)
-    data = content["data"]["productCreate"]
-    error = data["errors"][0]
-    assert error["field"] == "weight"
-    assert error["code"] == ProductErrorCode.INVALID.name
```

Comment:
What about checking the error code?
Same question for `field`. 
IMHO these tests should return the same error codes as previously. Checking the error msg is an additional validation but we should be sure that we didn't change the behavior of the API and we return the same errors as previously.